modules/ldap-syncing-running-all-ldap.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * authentication/ldap-syncing-groups.adoc
+
+[id="ldap-syncing-running-all-ldap_{context}"]
+= Syncing the LDAP server with {product-title}
+
+You can sync all groups from the LDAP server with {product-title}.
+
+.Prerequisite
+
+Create a sync configuration file.
+
+.Procedure
+
+. To sync all groups from the LDAP server with {product-title}:
```

Comment:
A procedure with only one step should be an unnumbered bullet, not a 1. A few other places.